### ISSUE TITLE
Support for linting sql.DB calls in methods

### ISSUE THREAD

**Issue by PiotrZakrzewski-instruqt** [ryanrolds/sqlclosecheck#32]
When linting methods calls to sqlx.DB in form of

```go
func (r *PostgresRepository) Method()  {
	stmt, err := r.db.Prepare("SELECT * FROM TABLE")
}
```

do not trigger sqlclosecheck. But the following does:

```go
func (r *PostgresRepository) Method()  {
        db := r.db
	stmt, err := db.Prepare("SELECT * FROM TABLE")
}
```

**Comment by PiotrZakrzewski-instruqt** [comment]
my bad. It was not flagged because the stmnt was used downstream
